src/neo-vm/ExecutionEngine.cs

```diff
@@ -1213,17 +1196,22 @@ private bool ExecuteInstruction()
             return true;
         }
 
+        protected virtual void LoadContext(ExecutionContext context)
+        {
+            if (EntryScriptHash is null)
+                EntryScriptHash = context.ScriptHash;
+            InvocationStack.Push(context);
+        }
+
         public ExecutionContext LoadScript(byte[] script, byte[] callingScriptHash = null, int rvcount = -1)
         {
             return LoadScript(new Script(Crypto, script), callingScriptHash, rvcount);
         }
 
-        protected virtual ExecutionContext LoadScript(Script script, byte[] callingScriptHash = null, int rvcount = -1)
+        private ExecutionContext LoadScript(Script script, byte[] callingScriptHash = null, int rvcount = -1)
         {
             ExecutionContext context = new ExecutionContext(script, callingScriptHash, rvcount);
```

Comment:
We can join this and remove this variable

Reply:
What do you mean?